src/vue/tokens-editor/actions-menu-token-edit.vue

```diff
@@ -3,35 +3,47 @@
          :style = "cssStyles"
     >
       <div class="alpheios-token-edit-actions-inner" ref="actionsInner">
-        <tooltip :tooltipText = "l10n.getMsgS('ACTION_BUTTON_UPDATE_TOKEN')" tooltipDirection = "top" v-if="allowedUpdateTokenWord">
-          <span class = "alpheios-token-edit-actions-button" 
-                id = "alpheios-token-edit-actions-button__update-token" 
-                @click = "$emit('updateTokenWord', token)"
-          >
-            <ok-icon />
-          </span>
-        </tooltip>
-        <span class = "alpheios-token-edit-actions-button alpheios-token-edit-actions-button__disabled" v-if="!allowedUpdateTokenWord"
-                id = "alpheios-token-edit-actions-button__update-token" 
-          >
-            <ok-icon />
-        </span>
+        <actions-button tooltipMess = "ACTION_BUTTON_UPDATE_TOKEN" :allowedCondition = "allowedUpdateTokenWord"
+                        actionName = "update_token" @click = "$emit('updateTokenWord', token)">
+          <template v-slot:enabled><ok-icon /></template>
+          <template v-slot:disabled><ok-icon /></template>

```

Comment:
I think it would be better to use a pencil icon rather than a checkmark for the edit action. Also, I notice that the color makes it always look disabled, and I think we need to have a different color for when it is active (i.e. when the user is in the process of editing)

Reply:
Updated to pen icon.

>Also, I notice that the color makes it always look disabled, and I think we need to have a different color for when it is active (i.e. when the user is in the process of editing)

Let divide issues here. I see two:

1. We have an option inside global options - Allow update token word. By default it is false. And that's why by default a user could only press space and arraws for splitting. So this icon is really disabled while it is false. If you choose to allow - it will become active as others icons.

![image](https://user-images.githubusercontent.com/12377640/104867236-27071080-598c-11eb-8cc7-11f025ac12b7.png)

2. About tracking the users actions inside input. I decided to reduce tracking events inside tokens editor to reduce resource usage. Because otherwise we would need to track key press for each input. That's why
- actions with tokens are made after menu button is clicked
- availability of actions are tracked only when token is updated (not the word inside token input)

And I suggest not to make the tracking process more difficult before we would have an ability to work with big texts by segments and would be able to tests it with it.

@balmas , does it make sense to you?

Reply:
Yes I think it's fine. I forgot about the option that was disabling the edit function. I think we will have some refinements to make here but agree it makes sense to wait until we have more test/user feedback.